src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void add(String fruit, int quantity) {
+        Storage.storage.put(fruit, quantity);
+    }
+
+    @Override
+    public int get(String fruit) {
+        return Storage.storage.get(fruit);
```

Comment:
Using `get()` directly on the map can result in a `NullPointerException` if the map contains no mapping for the key. It's safer to use `getOrDefault()` which allows you to provide a default value.